src/components/App.js

```diff
@@ -19,13 +19,21 @@ class App extends Component {
         {
           id: 2,
           coverArt: "image",
+          artist: "Bob Marley",
```

Comment:
Will this data always live in state from the jump? Or will state start empty, and only get populated after a fetch call to your backend?